proposals/Inflaiton Rate Change Modification Prop.md

```diff
@@ -0,0 +1,37 @@
+﻿
+Cosmos Hub Inflation variable modification: Inflation Rate Change
+
+**Quick Summary of issue**
+
+There are 6 main variables that control the maximum, minimum, & change of the atom inflation rate for the cosmos hub. These variables can be found inside the mint module via the attached hyperlink: [https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md](https://github.com/gavinly/CosmosParametersWiki/blob/master/Mint.md)
+
+In this proposal we will be looking at adjusting the inflation rate change variable.
+
+- When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year if a shock unbond occurred. Thusly, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.

```

Comment:
When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year **_after_** a shock unbond **_had_** occurred. **Thus**, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.

Reply:
Thanks for all the feedback - would you mind making the suggestions directly against the text? See
 https://haacked.com/archive/2019/06/03/suggested-changes/ for how to do that. And here's my example for this comment: https://github.com/cosmos/governance/pull/4/files#r528411005

Reply:
Thanks @ebuchman i was not able to fork and make a pull request for some reason and was unaware of the feature to make direct changes, thank you for the guidelines, appreciated.